sdk/ai/azure-ai-projects/azure/ai/projects/aio/operations/_patch.py

```diff
@@ -2456,7 +2456,7 @@ async def upload_file(
     async def upload_file_and_poll(self, *, body: JSON, sleep_interval: float = 1, **kwargs: Any) -> _models.OpenAIFile:
         """Uploads a file for use by other operations.
 
-        :keyword body: Required.
+        :param body: Required.
```

Comment:
nit: in this case body is a keyword as it is after `*`, so user have to say `body=""`

Reply:
updated to correspond with sync version